CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+# It uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+# Website
+packages/website/  @BMillman19 @fragosti
```

Comment:
💯

Reply:
Wanna add me to the list? I still maintain the developers section of the website :)

Reply:
sorry, didnt even see this, ill add you